docs/source/deduplication.rst

```diff
@@ -0,0 +1,42 @@
+System Deduplication and Merging
+================================
+Quipucords is used to inspect and gather information about your IT infrastructure.  System information can be gathered using the following types of sources:
+
+- network
+- vcenter
+- satellite
+
+A scan is composed of one or more sources.  A single system may be seen through multiple sources during a scan.  For example, a vcenter VM may be running a Satellite managed RHEL OS installation.  A network source could also be used to scan this system.  In this case, the system will be reported via vcenter, satellite, and network sources during a scan.  Quipucords feeds unprocessed system facts from a scan into a fingerprint engine.  The fingerprint engine matches and merges data for systems seen in more than one source.
```

Comment:
` A single system may be seen through multiple sources during a scan.  For example, a vcenter VM may be running a Satellite managed RHEL OS installation.`  
-->
` A single system may be found in multiple sources during a scan.  For example, a virtual machine on vCenter server may be running a Satellite managed RHEL OS installation.`